libs/datasets/timeseries.py

```diff
@@ -1433,7 +1464,7 @@ def combined_datasets(
     if static_series:
         output_static_df = pd.concat(
             static_series, axis=1, sort=True, verify_integrity=True
-        ).rename_axis(index=CommonFields.LOCATION_ID)
+        ).rename_axis(index=CommonFields.LOCATION_ID, columns=PdFields.VARIABLE)
```

Comment:
nit: Should we have the same Pandas 1.2 TODO here?

Reply:
I suspect it'll still be needed here because this DF is built from single columns. I changed the TODO to remind myself to search for all calls to rename_axis.